examples/mean/tests_mixedlang/test_mean.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import print_function
```

Comment:
I would avoid the same file in 2 places. 
Maybe better to have 3 makefiles (one for Verilog one for mixed and top that calls both)?

Reply:
Alternatively, set `PYTHONPATH=../tests` in this makefile, and then `MODULE` will point to the other python file.

Reply:
@eric-wieser Yes. Even better.

Reply:
OK, will do. thanks for the suggestion.